backend/src/api/sideBar/sideBar.controller.ts

```diff
@@ -46,13 +46,12 @@ export default class SideBarController {
 
     public async getAllSidebar(): Promise<ServiceResponse<SideBarOptions[] | null>> {
         try {
-            const sidebarOptionsSet: Set<string> = new Set<string>()
+            let blocksData: FullTemplate[] = await FullTemplateModel.find({})
 
-            const blocksData = this.readBlocksData()
```

Comment:
Was this done in #96? Perhaps you've forgotten to rebase from the latest version of develop